pkgs/development/ocaml-modules/minttea/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildDunePackage
+, fetchurl
+, mdx
+, riot
+, tty
+}:
+
+buildDunePackage rec {
+  pname = "minttea";
+  version = "0.0.1";
+
+  minimalOCamlVersion = "5.1";
+
+  src = fetchurl {
+    url = "https://github.com/leostera/minttea/releases/download/${version}/minttea-${version}.tbz";
+    hash = "sha256-+4nVeYKx2A2i2nll/PbStcEa+Dvxd0T7e/KsdJqY4bI=";
+  };
+
+  propagatedBuildInputs = [
+    mdx
```

Comment:
```suggestion
```

The `mdx` library should not be propagated.

Are you sure it is even needed at build time?

Reply:
Good point -- I think this is only required for testing. Let me remove this.

Reply:
OK, I removed `mdx`. Looks like I can build without it!